templates/options.conf.erb

```diff
@@ -4,6 +4,11 @@ forwarders { <%= scope.lookupvar('::dns::forwarders').join("; ") %>; };
 <% end -%>
 
 listen-on-v6 { <%= scope.lookupvar('::dns::listen_on_v6') %>; };
+
+<% unless scope.lookupvar('::dns::allow_recursion').empty? -%>
```

Comment:
This doesn't allow to fully disable recursion, which IMHO is a missing feature right now. IMHO it would require a major version bump, but I'd be in favour of disabling recursion by default. The internet has become a hostile place and open recursors are a big problem due to reflection attacks.